entity-service/src/main/resources/configs/common/application.conf

```diff
@@ -78,3 +79,14 @@ metrics.reporter {
   console.reportInterval = 30
 }
 
+event.store {
+  type = kafka
+  bootstrap.servers = "localhost:9092"
+  entity.change.events.producer {
+    topic.name = entity-change-events
+    bootstrap.servers = "localhost:9092"
+    key.serializer = org.apache.kafka.common.serialization.StringSerializer
+    value.serializer = io.confluent.kafka.serializers.protobuf.KafkaProtobufSerializer
+    schema.registry.url = "http://localhost:8081"
```

Comment:
A little late to the review but I'm assuming `schema.registry.url` is not needed. The schema registry is specific to avro messages but here its proto.

Reply:
Actually schema registry also supports protos. We are using version confluent version 5.5 which does support protos schema.